website/content/api-docs/index.mdx

```diff
@@ -21,12 +21,6 @@ is, `sys/` routes) change very infrequently, but various secrets engines/auth
 methods/etc. sometimes have minor changes to accommodate new features as
 they're developed.
 
```

Comment:
The request PATH difference seems to be one of the most obvious differences between v1 and v2, but this change seem to be dropping this bit of information all together?

Reply:
@benashz I've added a specific note on URI path differences and features.